samples/v1/language_classify_gcs.js

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// DO NOT EDIT! This is a generated sample ("Request",  "language_classify_gcs")
+
+// sample-metadata:
```

Comment:
Thank you for including these!

Reply:
I love them! I parse and use it in a separate CLI tool for easily running samples. So glad we have this, we can generate READMEs across langs with them 💕